jf_agent/git/github_gql_client.py

```diff
@@ -169,21 +169,34 @@ def get_raw_result(self, query_body: str) -> dict:
                     f'A secondary rate limit was hit. Sleeping for {sleep_time} seconds. (attempt {attempt_number}/{max_attempts})',
                 )
                 time.sleep(sleep_time)
-                attempt_number += 1
             except GqlRateLimitedException:
+                if attempt_number > max_attempts:
+                    raise
+
                 rate_limit_info = self.get_rate_limit(base_url=self.base_url)
                 reset_at: datetime = github_gql_format_to_datetime(rate_limit_info['resetAt'])
-                reset_at_timestamp = reset_at.replace(tzinfo=timezone.utc).timestamp()
+                reset_at_timestamp = reset_at.timestamp()
```

Comment:
The `.replace(tzinfo=timezone.utc)` call is already made as part of the `github_gql_format_to_datetime`. This is just a little cleanup